### PR TITLE
Utility to get config from json

### PR DESCRIPTION
**Pull request by rafaelfolco**
It reads a specific part of the json file and dumps the inner
config to stdout.

### CODE REVIEW

File: bin/crucible-run.json.example [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                    { "arg": "bs", "vals": [ "4K", "16K" ] }
+                ]
+            }
+        ]
+    },
+    "tool-params": {
+    },
+    "passthru-args": ""
+}
+
```
Comment (k-rister) [inline bin/crucible-run.json.example:31]: Again, is the `bin` directory really the best place for this?

File: bin/get-json-config.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
```
Comment (k-rister) [inline bin/get-json-config.py:1]: I think we need to be careful to either keep this tool separated from this one:

https://github.com/perftool-incubator/toolbox/blob/main/bin/get-json-settings.pl
https://github.com/perftool-incubator/toolbox/blob/main/perl/toolbox/jsonsettings.pm

Or figure out how to combine them.

The existing tool is used to process a file like this:

https://github.com/perftool-incubator/rickshaw/blob/master/rickshaw-settings.json

In ways such as these:

https://github.com/perftool-incubator/rickshaw/blob/master/rickshaw-run#L1393-L1441

https://github.com/perftool-incubator/rickshaw/blob/master/endpoints/base#L920-L958

https://github.com/perftool-incubator/rickshaw/blob/master/engine/engine-script-library#L1090-L1109

https://github.com/perftool-incubator/rickshaw/blob/master/engine/engine-script-library#L271

Comment (k-rister) [reply bin/get-json-config.py:1]: I think we need to create either an examples directory or a doc directory, something like that.

File: bin/crucible-run.json.example (hunk 1 above)
Comment (rafaelfolco) [reply bin/crucible-run.json.example]: ack

File: bin/get-json-config.py (hunk 2 above)
Comment (rafaelfolco) [reply bin/get-json-config.py:1]: done

**Review by k-rister — APPROVED** [review]

**Pull request merged**